src/middleware/AppContext.tsx

```diff
@@ -66,6 +77,8 @@ export const ContextProvider = ({ children }: { children: ReactNode }) => {
         rbac: rbac,
         features: features,
         isFetchingFeatures: isFetchingFeatures,
+        subscriptions: subscriptions,
+        isFetchingSubscriptions: isFetchingSubscriptions,
```

Comment:
We can likely combine isFetchingFeatures and isFetchingSubscriptions into something like:
"isFetchingPermissions" for use downstream.
No need for two variables.

Reply:
good point 👍🏼 resolved [here](https://github.com/content-services/content-sources-frontend/pull/328/commits/92ada247ce4e2cfe19fb4e6adac241f9ae08e54d) 🫡